pkg/util/globalerror/errors.go

```diff
@@ -84,5 +84,16 @@ func (id ID) MessageWithLimitConfig(msg, flag string, addFlags ...string) string
 		sb.WriteString(" and -")
 		sb.WriteString(addFlags[len(addFlags)-1])
 	}
-	return fmt.Sprintf("%s (%s%s). To adjust the related per-tenant limit%s, configure %s, or contact your service administrator.", msg, errPrefix, id, plural, sb.String())
+	return fmt.Sprintf(template, msg, errPrefix, id, plural, sb.String())
+}
+
+// MessageWithLimitConfig returns the provided msg, appending the error id and a suggestion on
+// which configuration flag(s) to use to change the limit.
+func (id ID) MessageWithLimitConfig(msg, flag string, addFlags ...string) string {
+	return id.MessageWithLimitConfigBuilder("%s (%s%s). To adjust the related per-tenant limit%s, configure %s, or contact your service administrator.", msg, flag, addFlags...)
+}
+
+// MessageWithGlobalLimitConfig is like MessageWithLimitConfig but for non-per-tenant limits
+func (id ID) MessageWithGlobalLimitConfig(msg, flag string, addFlags ...string) string {
```

Comment:
Can you rebase `main`, please? We introduced `MessageWithPerInstanceLimitConfig()` for limits which apply to the instance (replica). Use `MessageWithPerInstanceLimitConfig()` instead of `MessageWithGlobalLimitConfig()` (so far the global naming is about limits whose value applies to a value computed on top of the whole cluster).